### DISCUSSION TITLE
General opinon on v0.9.0

### DISCUSSION THREAD

**Discussion by FerasK0** [robertodoering/harpy#646] [General]
I'm gonna state my opinion on the execution or status of certain fixes/features

and all of the other alpha testers are welcome to participate!

1- #600 , #603 , #574 is fixed and working perfectly fine

2- #479 the new media player is awesome, but kinda bland when you're tweeting, but honestly not bad for a beta 👍 

3- #487 the over scroll is a nice addition 

4- #526 could use some improvements, for example, making the followers count above the pressed area rather than the bottom to increase visibility, and putting commas to make it easier to read (9.840.134M is better than 9840134)

5- #379 doesn't work after playing the stopped video.

**Comment by monee121** [comment]
**#498** - Ability to search followers/following from those lists in your profile would be nice. Like a search bar at the top so you can find someone you follow. Or auto populate followers/following first from the general search bar. Not sure which option works best from the dev side. 

**#643** - happens to me also and fixes itself with a scroll up and down

**Comment by dontobi** [comment]
What about an update for the alpha? Then we could still test the last fixes. So far I haven't noticed any other problem.

**Reply by robertodoering** [reply]
Thanks for testing! An update is currently in review in the play store. If everything goes right I will also push this update to beta on Thursday.

**Reply by dnestico** [reply]
Nice!

Will it have a changelog so we know what to test by the way? I haven't updated to alpha yet but will tomorrow so would be nice to know.

**Reply by FerasK0** [reply]
> Nice!
> 
> Will it have a changelog so we know what to test by the way? I haven't updated to alpha yet but will tomorrow so would be nice to know. 

About the changelog, it's only present in the free version, the pro is just blank, so don't forget to fix it :) 

But honestly, the app feels fresh and stable (for now at least) 


Pro:
. ![Screenshot_20220518-080657_harpy.png](https://user-images.githubusercontent.com/83197825/168961381-457b0fdb-6245-4889-9669-ec98b4687d9f.png)

Free:
![Screenshot_20220518-080702_harpy free.png](https://user-images.githubusercontent.com/83197825/168961408-4b8e7063-19c4-42fb-9b84-0639ce4f242b.png)

**Reply by dnestico** [reply]
Just updated and the app defenitley runs smoother and better!

**Reply by robertodoering** [reply]
Nice to hear! I have some jankyness when a list is scrolled down far and I switch tabs (e.g. scroll horizontally). Does the app also stutter for you in that case?
I'm contemplating whether it makes sense to not remember the scroll position (or to make it optional) to avoid this issue for now, what do you think of that?

**Reply by dontobi** [reply]
For me, Harpy 0.9.0 runs smoother than the old version. However, I can confirm your impression when scrolling and switching tabs.

**Reply by dnestico** [reply]
Harpy runs way smoother and loads up instantly.

But the remember scroll position option does not seem to be working for me right now. 

Hope it's not removed as I really like it remembering my position.

**Reply by robertodoering** [reply]
Thanks, that was indeed not working as intended but I fixed it now
